webpack.common.ts

```diff
@@ -96,6 +96,11 @@ const config: webpack.Configuration = {
           mimetype: 'image/svg+xml',
         },
       },
+      {
+        test: /\.(graphql|gql)$/,
```

Comment:
This is so when we import from `.graphql` files, they actually get embedded in the app as strings